### PR TITLE
GH-1664 DDL Choro Layers

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-1664

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage @jchambers-ln Please Review

File: test/mapFactory.js [hunk 1]
```diff
@@ -183,6 +183,22 @@
...
             },
+            pins: function (callback) {
+                require(["test/DataFactory", "src/map/ChoroplethStates", "src/map/Pins"], function (DataFactory, ChoroplethStates, Pins) {
+                    var pins = new Pins()
+                        .columns(DataFactory.GMap.simple.columns)
+                        .data(DataFactory.GMap.simple.data)
+                    ;
+                    var choro = new ChoroplethStates()
+                        .columns(["State", "Weight"])
+                        .data([["AL", 4779736], ["FL", 710231]])
```
Comment (bwood1) [inline test/mapFactory.js]: Is this meant to be populated with hard coded data?

**Comment by bwood1** [comment]
One comment and this has a merge conflict. Other than that it looks good.

File: test/mapFactory.js (hunk 1 above)
Comment (GordonSmith) [inline test/mapFactory.js]: Yes - my rule of thumb is to include the data inline _unless_ it is needed in two diff tests, then I move it into the DataFactory.

**Pull request merged**